services/common/src/components/projectSummary/ProjectSubmissionStatusPage.tsx

```diff
@@ -0,0 +1,133 @@
+import * as API from "@mds/common/constants/API";
+import { AMS_STATUS_CODE_FAIL } from "@mds/common/constants";
+
+export const ProjectSubmissionStatusPage = () => {
+  const { projectGuid, status } = useParams<{ projectGuid: string; status: string }>();
+
+  const downloadIRTTemplate = (url) => {
+    const anchor = document.createElement("a");
+    anchor.href = url;
+    anchor.style.display = "none";
+    document.body.append(anchor);
+    anchor.click();
+    anchor.remove();
+  };
+
+  const renderContent = () => {
+    let content = (
+      <div style={{ textAlign: "center" }}>
+        <>
+          <Row>
+            <Col span={24}>
+              <CheckCircleOutlined className="success-page" />
+            </Col>
+          </Row>
+          <br />
+          <Typography.Paragraph>
+            <Typography.Title level={4}>
+              Thank you, your application has been submitted successfully!
+            </Typography.Title>
+            You can track the submission status in Project Description Overview.
+          </Typography.Paragraph>
+          <Typography.Paragraph>
+            If your application requires{" "}
+            <a
+              href="https://www2.gov.bc.ca/gov/content/industry/mineral-exploration-mining/permitting/coordinated-authorizations"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Information Requirements Table (IRT)
+            </a>
+            ,
+          </Typography.Paragraph>
+          <Typography.Paragraph>
+            download{" "}
+            <LinkButton
+              onClick={() =>
+                downloadIRTTemplate(
+                  ENVIRONMENT.apiUrl + API.INFORMATION_REQUIREMENTS_TABLE_TEMPLATE_DOWNLOAD
+                )
+              }
+            >
+              IRT template
+            </LinkButton>{" "}
+            for the next phase.
+          </Typography.Paragraph>
+          <div>
+            <p>
+              <Link to={GLOBAL_ROUTES?.EDIT_PROJECT.dynamicRoute(projectGuid)}>
+                <Button type="primary">Back to Project Overview</Button>
+              </Link>
+            </p>
+            <p>
+              <Link to={{}}>
+                <Button>View Project Description Overview</Button>
+              </Link>
+            </p>
+          </div>
+        </>
+      </div>
+    );
+
+    if (status === AMS_STATUS_CODE_FAIL) {
+      content = (
+        <div style={{ textAlign: "center" }}>
+          <>
+            <Row>
+              <Col span={24}>
+                <CloseCircleOutlined className="fail-page" />
+              </Col>
+            </Row>
+            <br />
+            <Typography.Paragraph>
+              <Typography.Title level={4}>Submission Failed.</Typography.Title>
+              Please retry submission in Project Description Overview in a few minutes.
+            </Typography.Paragraph>
+            <div>
+              <p>
+                <Link to={{}}>
```

Comment:
This link is empty at the moment but will redirect to the project description overview page that Henry is working on.